mindsdb_native/libs/controllers/predictor.py

```diff
@@ -263,7 +263,8 @@ def learn(self,
                 allow_incomplete_history = advanced_args.get('allow_incomplete_history', False),
                 quick_learn = advanced_args.get('quick_learn', None),
                 quick_predict = advanced_args.get('quick_predict', False),
-                apply_to_columns = advanced_args.get('apply_to_columns', {})
+                apply_to_columns = advanced_args.get('apply_to_columns', {}),
+                disable_column_importance = advanced_args.get('disable_column_importance', {})
```

Comment:
since it is a boolean parameter I think that default value should be `False` not `{}`